docs/install.md

```diff
@@ -0,0 +1,84 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+# Hyperledger Private Data Objects, Sample Contracts #
+
+The following instructions assume that the PDO contracts source is available in the directory pointed to by the environment variable `PDO_CONTRACTS_ROOT` and will be installed to the directory pointed to by the environment variable `PDO_INSTALL_ROOT`.
+
+## Process Overview
+
+- Install required build dependencies
+- Install the WASM development toolchain
+- Build and install the PDO client environment
+- Build and install the PDO contracts packages
+
+## <a name="environment">Install Build Dependencies</a>
+
+On a minimal Ubuntu system, the following packages are required. Other
+distributions will require similar packages.
+
+```
+sudo apt install -y cmake curl git pkg-config unzip xxd libssl-dev build-essential
+sudo apt install -y swig python3 python3-dev python3-venv virtualenv
+sudo apt install -y liblmdb-dev libprotobuf-dev libsecp256k1-dev protobuf-compiler libncurses5-dev
```

Comment:
are these anything more than what a pdo-client already needs ? Otherwise, it be good to clarify that.

Reply:
this list is precisely what is recommended for PDO client. we can drop several of these, but this is sufficient for now.